public/images/double-arrows-select-icon.tsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+const DoubleArrowsSelectIcon = () => (
+  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 7 12" width="7px">
+    <g fill="#CFCFF5" fillRule="evenodd">
```

Comment:
Because I needed to update this fill with CSS, I'm essentially importing the SVG as a component so that I can use it inline rather than referenced through an img tag. We could do this without manually converting the SVG pretty easily in our create-react-app. Is there a better way to do this in Next?

Reply:
Hmm yeah, how does it work in CRA? If it's with a Webpack loader we could probably add that loader to our `next.config.js` similar to what `withImages` does.

We may also be able to use the [`css` prop](https://emotion.sh/docs/css-prop#string-styles) here or pass the `fill` in as a prop?

---

Somewhat related: I think we started by putting stuff in `/public` since Next.js serves that publicly (allowing `<img src="/my-image.png" />`, but it doesn't work as well with Storybook since it does everything with Webpack. Probably a larger discussion for the team, but maybe we start putting these in `components/atoms` and make a ticket to move what we can from `/public`.

Reply:
CRA lets you import them as components by changing the import statement from `import logo from './logo.svg';` to `import { ReactComponent as Logo } from './logo.svg';` (per [here](https://create-react-app.dev/docs/adding-images-fonts-and-files/#adding-svgs)). I'm not sure how they do it behind the scenes

I looked at maybe doing something like [this](https://github.com/airbnb/babel-plugin-inline-react-svg), but that would change all of our SVGs to component imports and prevent us from using them in img tags (which we're already doing in a number of places)

Reply:
Ah yeah those look interesting. I wouldn't be opposed to switching the existing SVGs to use something different so maybe we do that in another ticket after some more research?

Reply:
Sounds like a plan! I created https://dwolla.atlassian.net/browse/WEB-670 to track it